sops/3scale-scaling.md

```diff
@@ -0,0 +1,71 @@
+# 3Scale Scaling Guide
+The purpose of this guide is to outline the existing scaling capabilities of 3Scale, including scaling options for high availability and performance driven implementations along with some known limitations within the product.
+
+**Note :** Pod [anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) rule is in place across a number of 3scale components. This may result in limited scaling options.
+
+## Component Scaling Options
+**Note :** Components are ordered in priority of known bottle necks.
+
+
+1. Backend-Listener
+    - Pod **anti-affinity** rule is important here
+    - You can scale this component **horizontally** by adding more PODs
+        - `oc scale dc backend-listener -n <ns-prefix>-3scale --replicas=<number of replicas>`
+1. Backend-Worker
+    - Pod **anti-affinity** rule is important here
+    - Critical functionality -- rate limits depend on this component
+    - Depending on the number of reports you should check the lenght of the redis queues for jobs
+    - Can be scaled **horizontally** by adding more PODs as needed.
+        - `oc scale dc backend-worker -n <ns-prefix>-3scale --replicas=<number of replicas>`
```

Comment:
To match up with the <ns-prefix> example we should change <number of replicas> to <number-of-replicas>